README.md

```diff
@@ -1,2 +1,70 @@
-# bookstore-react-redux
-The Bookstore is a website that allows you to display a list of books, add a book, remove a selected book.
+# Getting Started with Create React App
+
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
```

Comment:
- [ ] Your README is a dummy file. Kindly format it to be more descriptive using the Microverse Template [here](https://github.com/microverseinc/readme-template).
- [ ] Also remember to include how users can set up your app locally and install packages (`npm install`) and run the app (`npm start`)